data/json/recipes/recipe_others.json

```diff
@@ -3704,11 +3702,26 @@
     "subcategory": "CSC_OTHER_MATERIALS",
     "skill_used": "survival",
     "difficulty": 1,
-    "time": "20 m",
+    "time": "20 s",
     "autolearn": true,
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "cured_pelt", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "cured_hide",
```

Comment:
I can't find a resource that indicates that drying via heating is a workable method for preserving hide.